pontoon/base/templatetags/helpers.py

```diff
@@ -371,6 +371,6 @@ def set_attrs(attrs, new=False):
         return attrs
 
     # Escape all tags
-    linker = Linker(callbacks=[set_attrs], recognized_tags=[])
+    linker = Linker(callbacks=[set_attrs])
```

Comment:
We shouldn't have used [recognized_tags](https://bleach.readthedocs.io/en/latest/linkify.html#using-bleach-linkifier-linker) in first place.